mods/ts/rules/defaults.yaml

```diff
@@ -566,6 +569,9 @@
 	Carryable:
 		RequiresCondition: !inside-tunnel
 	RevealOnFire:
+	RejectsOrders@tunnel:
+		Reject: Stop
```

Comment:
`Move.Cancel` is also called from the `MovePart`s' `Cancel` methods, and they're using the `path` via their `parent` argument.